Assets/Common/AddressRecords.json

```diff
@@ -4,6 +4,21 @@
         "Pattern": "48 83 EC 48 48 8B 05 ? ? ? ? 4C 8D 0D ? ? ? ? BA 0A 00 00 00",
         "Offset": 0
     },
+    {
+        "Name": "Core::ScrtCommonMain",
+        "Pattern": "48 89 5C 24 08 57 48 83 EC 30 B9 01 00 00 00 E8 ? ? ? ? 84 C0",
+        "Offset": 0
+    },
+    {
+        "Name": "Core::WinMainCall",
+        "Pattern": "E8 ? ? ? ? 0F B7 D8 E8 ? ? ? ? 4C 8B C0 44 8B CB 33 D2 48 8D ? ? ? ? ?",
+        "Offset": 28
+    },
+    {
+        "Name": "Core::MhMainCtor",
+        "Pattern": "BA 00 00 08 00 48 8B CF E8 ? ? ? ? 4C 89 3F C7 87 ? ? ? ? FF FF FF FF",
+        "Offset": -122
+    },
```

Comment:
Don't bother changing it, but usually I use a single `:` for separating Type:Method in this file. Not sure why I picked that but yeah, for future reference 😆